src/common/framecmn.cpp

```diff
@@ -343,7 +343,15 @@ void wxFrameBase::OnMenuHighlight(wxMenuEvent& event)
 {
     event.Skip();
 
-    (void)ShowMenuHelp(event.GetMenuId());
+    wxMenuItem* menuItem = event.GetMenuItem();
+    if( menuItem )
```

Comment:
Minor, but I'd write this as
```suggestion
    if ( wxMenuItem* menuItem = event.GetMenuItem() )
```